models/model_upload/README.md

```diff
@@ -94,6 +97,8 @@ hf_token: "your_hf_token" # Required for private models
 
 ### Step 2: Define dependencies in requirements.txt
 List all required Python dependencies for your model in `requirements.txt` file. This ensures that all necessary libraries are installed in the runtime environment
+Make sure to include `clarifai` dependency in the `requirements.txt` too for the next step.
+If you follow strict versioning, pick latest version from [pip](https://pypi.org/project/clarifai/)
```

Comment:
Why do we need to include the `clarifai` dependency in `requirements.txt`? We don't need it because the Dockerfile templates already have a command to install Clarifai in the container during model image building